packages/calcite-components/src/components/action-menu/action-menu.scss

```diff
@@ -1,17 +1,12 @@
 :host {
-  @apply text-color-2
-  text-1
+  @apply text-1
   box-border
   flex
   flex-col;
 }
 
 ::slotted(calcite-action-group) {
-  border-block-end: 1px solid var(--calcite-color-border-3);
-}
-
-::slotted(calcite-action-group:last-child) {
```

Comment:
question: Wouldn't removing this this make the last group have a border-bottom? Can you verify this isn't breaking anything?

Reply:
I handed back control of calcite-action-group to itself. The style you're looking for is on line 27 of action-group.scss

Reply:
Thanks